components/server/src/server.rs

```diff
@@ -256,6 +258,14 @@ impl<ER: RaftEngine> TiKVServer<ER> {
         let latest_ts = block_on(pd_client.get_tso()).expect("failed to get timestamp from PD");
         let concurrency_manager = ConcurrencyManager::new(latest_ts);
 
+        let quota_limiter = Arc::new(QuotaLimiter::new(
+            config.quota.cpu,
+            config.quota.write_kvs,
```

Comment:
For simplicity, let's remove write_kvs, total_qps  for now.

Reply:
finished